cli/tsc_config.rs

```diff
@@ -52,37 +52,42 @@ impl fmt::Display for IgnoredCompilerOptions {
 /// A static slice of all the compiler options that should be ignored that
 /// either have no effect on the compilation or would cause the emit to not work
 /// in Deno.
-const IGNORED_COMPILER_OPTIONS: [&str; 10] = [
+const IGNORED_COMPILER_OPTIONS: [&str; 20] = [
   "allowSyntheticDefaultImports",
+  "declaration",
+  "declarationMap",
+  "downlevelIteration",
   "esModuleInterop",
+  "emitDeclarationOnly",
+  "importHelpers",
   "inlineSourceMap",
   "inlineSources",
   // TODO(nayeemrmn): Add "isolatedModules" here for 1.6.0.
   "module",
+  "noEmitHelpers",
   "noLib",
   "preserveConstEnums",
   "reactNamespace",
+  "skipDefaultLibCheck",
+  "skipLibCheck",
   "sourceMap",
   "target",
+  "types",
+  "useDefineForClassFields",
 ];
 
-const IGNORED_RUNTIME_COMPILER_OPTIONS: [&str; 50] = [
+const IGNORED_RUNTIME_COMPILER_OPTIONS: [&str; 40] = [
```

Comment:
Consider changing type to `&[&str]`, then you won't have to specify number of elements with each change.